zoo-css-challenge/index.html

```diff
@@ -1,136 +1,208 @@
+        <h4>2021-05-26</h4>
+        <h4>Which big cat are you?</h4>
+        <div class="card">
+          <time>2010-04-16</time>
+          <h4>Helping wild turtles in Seychelles</h4>
+          <a>Find out More</a>
+        </div>
+        <div class="card">
+          <time>2010-04-12</time>
+          <h4>Learning about the rainforest</h4>
+          <a>Find out More</a>
+        </div>
+      </section>
 
-		<section class="education">
-			<div class="education-container">
-				<h3>Learning resources for schools</h3>
-				<p>Explore our activities, videos, and lesson plans to support a wide range of curriculum topics</p>
-				<button>Find them here</button>
-			</div>
-		</section>
+      <section class="education">
+        <div class="education-container">
+          <h3>Learning resources for schools</h3>
+          <p>
+            Explore our activities, videos, and lesson plans to support a wide
+            range of curriculum topics
+          </p>
+          <button>Find them here</button>
+        </div>
+      </section>
 
-		<section class="membership">
-			<h2>Are there any benefits or discounts for members?</h2>
-			<p>There are <strong>AMAZING</strong> benefits of becoming a member with Birmingham Zoo! Our members get special discounts in our shops, cafes, and unlimited entry</p>
-			<button>Learn More</button>
-		</section>
+      <section class="membership">
+        <h2>Are there any benefits or discounts for members?</h2>
+        <p>
+          There are <strong>AMAZING</strong> benefits of becoming a member with
+          Birmingham Zoo! Our members get special discounts in our shops, cafes,
+          and unlimited entry
+        </p>
+        <button>Learn More</button>
+      </section>
 
-		<section class="programs">
-			<h2>Programs</h2>
-			<div class="card">
-				<h4>Children and Family Programs</h4>
-				<a>Find out More</a>
-			</div>
-			<div class="card">
-				<h4>Adult Programs</h4>
-				<a>Find out More</a>
-			</div>
-			<div class="card">
-				<h4>Youth and young adults</h4>
-				<a>Find out More</a>
-			</div>
-			<h3>Useful Links</h3>
-			<p>
-			</p>
-			<a href="https://www.w3schools.com">w3schools.com</a>
-			<a href="https://www.nationalgeographic.com/" target="_blank">National Geographic</a>
-			<a href="http://www.wikipedia.org" target="_top">Wikipedia</a>
-		</section>
-
- 	</main>
- 	<footer>
- 		<p>Open Monday-Friday 7am-4pm</p><address>1 Birmingham Road</address>
- 	</footer>
+      <section class="programs">
+        <h2>Programs</h2>
+        <div class="card">
+          <h4>Children and Family Programs</h4>
+          <a href="https://www.w3schools.com/">Find out More</a>
+        </div>
+        <div class="card">
+          <h4>Adult Programs</h4>
+          <a href="https://www.w3schools.com/">Find out More</a>
+        </div>
+        <div class="card">
+          <h4>Youth and young adults</h4>
+          <a href="https://www.w3schools.com/">Find out More</a>
+        </div>
```

Comment:
Your cards are still stacked vertically. To have your cards side by side,
wrap your cards in `<div class="card-container">` like this:

```
        <div class="card-container">
          <div class="card">
            <h4>Children and Family Programs</h4>
            <a href="https://www.w3schools.com/">Find out More</a>
          </div>
          <div class="card">
            <h4>Adult Programs</h4>
            <a href="https://www.w3schools.com/">Find out More</a>
          </div>
          <div class="card">
            <h4>Youth and young adults</h4>
            <a href="https://www.w3schools.com/">Find out More</a>
          </div>
        </div>
```
In your css file, add this code:
```
.programs .card-container {
  width: 100%;
  display: flex;
  flex-direction: row;
}
```